WasatchNET/MultiChannelWrapper.cs

```diff
@@ -340,10 +342,16 @@ public async void startAcquisition()
         /// total acquisition time is close to minimal.
         /// </remarks>
         /// <param name="sendTrigger">whether to automatically raise the HW trigger (default true)</param>
-        public List<ChannelSpectrum> getSpectra(bool sendTrigger=true)
+        public async Task<List<ChannelSpectrum>> getSpectra(bool sendTrigger=true)
         {
             if (sendTrigger)
-                startAcquisition();
+            {
+                if (!await startAcquisition())

```

Comment:
This was the fundamental bug which Cullon uncovered in his MultiChannel testing.  I wasn't actually waiting on the trigger pulse to complete (rise, level, then fall) before moving on to reading the spectra whose integrations were presumably started by the pulse.

Reply:
I assume this goes along with the changes to Form1? I'll need to dig in once I start doing async await in WPSC